experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -379,6 +386,7 @@ export class SharedTree
 		local: boolean,
 		localOpMetadata: unknown,
 	) {
+		this.currentSeq = message.sequenceNumber;
```

Comment:
this was possibly discussed elsewhere, but why don't we just use the getter on the runtime in shared object? is it not set until after op processing?

Reply:
Updated to use the latest sequence number from the runtime in shared object.

Reply:
Did you confirm it returned the same value as your implementation?